pkg/network/node/node.go

```diff
@@ -31,7 +31,7 @@ import (
 	kinternalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
 	kubeletapi "k8s.io/kubernetes/pkg/kubelet/apis/cri"
 	kruntimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
-	knetwork "k8s.io/kubernetes/pkg/kubelet/network"
+	knetwork "k8s.io/kubernetes/pkg/kubelet/dockershim/network"
```

Comment:
Actually, this import (in pkg/network/node/node.go) can be removed, and the two `OsdnNode` field members that use it can be removed. They're just cruft.

Other that that, `DANW: pre-rebase package cleanup` looks right

Reply:
> Actually, this import (in pkg/network/node/node.go) can be removed, and the two OsdnNode field members that use it can be removed. They're just cruft.
> 
> Other that that, DANW: pre-rebase package cleanup looks right

fixed